lib/driver-helper.js

```diff
@@ -253,6 +253,23 @@ export function unsetCheckbox( driver, selector ) {
 	} );
 }
 
+export function waitTillPresent( driver, selector, waitOverride ) {
```

Comment:
We already have the function `waitTillPresentAndDisplayed` which appears to do the same thing. I swapped it in to test and it works too. Is there a reason to have a different one or was it just overlooked?

Reply:
You're right! This was overlooked from my first approach using a hidden input element when I needed to know if it was only present and not displayed, which would have returned false. Since this isn't needed anymore I'll push an update removing this function.